### PR TITLE
[API-699] Clean up documentation for segments and streams endpoints

### PR DESCRIPTION
**Pull request by zackisaacs**
I went through and cleaned up the documentation for each of the segments and streams endpoints. I also added sample json responses for each endpoint. For this review, I'm primarily looking for grammatical errors and any personal data that I failed to sanitize properly.

Completes [API-699](https://strava.atlassian.net/browse/API-699)

### CODE REVIEW

File: swagger/swagger.json.mustache [hunk 1]
```diff
@@ -189,13 +189,61 @@
         ],
         "responses": {
           "200": {
-            "description": "Representation of a segment",
+            "description": "Representation of a segment,",
```
Comment (hariniriyer) [inline swagger/swagger.json.mustache]: Should there be a '.' instead?

File: swagger/swagger.json.mustache [hunk 2]
```diff
@@ -189,13 +189,61 @@
...
             "schema": {
               "$ref": "{{reference_prefix}}/segment.json#/DetailedSegment"
+            },
+            "examples": {
+              "application/json": {
+                "id": 229781,
```
Comment (hariniriyer) [inline swagger/swagger.json.mustache]: This is a real segment. So I am assuming that it is something that is public and okay to expose.

Comment (zackisaacs) [reply swagger/swagger.json.mustache]: Thanks for checking. I think exposing this segment is OK. Hawk Hill is one of the most famous segments on Strava, and I scrubbed all personal data from it.

File: swagger/swagger.json.mustache (hunk 1 above)
Comment (zackisaacs) [reply swagger/swagger.json.mustache]: Yep, thanks!

File: swagger/swagger.json.mustache [hunk 3]
```diff
@@ -189,13 +189,61 @@
         ],
         "responses": {
           "200": {
-            "description": "Representation of a segment",
+            "description": "Representation of a segment.",
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: `,` => `.`

**Comment by zackisaacs** [comment]
friendly ping

**Pull request merged**